packages/datadog-plugin-iovalkey/src/index.js

```diff
@@ -0,0 +1,11 @@
+'use strict'
+
+const RedisPlugin = require('../../datadog-plugin-redis/src')
+
+class IOValkeyPlugin extends RedisPlugin {
```

Comment:
By extending from the Redis plugin, a couple of things are hardcoded to `redis`. This is probably not ideal or should we just accept it being reported as redis? That might be confusing long term?

I guess we could start with it and change it later on as well.